### DISCUSSION TITLE
Preparation for Pypi 2FA requirements

### DISCUSSION THREAD

**Discussion by wynquinox** [grpc-requests/grpc_requests#20] [General]
@wesky93 - raising this out of curiousity.  Is the maintainer account for the GRPC package prepared for the 2FA requirements that were announced on 2023-05-25?

https://blog.pypi.org/posts/2023-05-25-securing-pypi-with-2fa/

**Comment by wesky93** [comment]
I have completed 2FA enrollment for my account.
Thank you!

**Reply by wesky93** [reply]
@ViridianForge Can you tell me your registered account on pipy? I'd like to invite you to be a maintainer on grpc_requests.

**Reply by wynquinox** [reply]
I'm ViridianForge over there as well, thanks!

**Reply by wesky93** [reply]
I invited you!

**Reply by wynquinox** [reply]
Accepted, thanks dude!
